daheinze-oscheer/pointersanderrors/wallet.go

```diff
@@ -0,0 +1,46 @@
+package pointersanderrors
+
+import (
+	"errors"
+	"fmt"
+)
+
+// Stringer lorem ipsum
+type Stringer interface {
+	String() string
+}
```

Comment:
Do we still need this?

Reply:
not really, as no other "class" is using this, it is not necessary to define an interface here. I removed it.